streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/config/Config.java

```diff
@@ -1,11 +1,10 @@
 package io.ray.streaming.runtime.config;
 
-import java.io.Serializable;
-import javax.accessibility.Accessible;
+import org.aeonbits.owner.Accessible;
 
 /**
  * Basic config interface.
  */
-public interface Config extends org.aeonbits.owner.Config, Accessible, Serializable {
+public interface Config extends org.aeonbits.owner.Config, Accessible {
```

Comment:
Why remove ```Serializable```, configuration needs  to be passed across the network.

Reply:
Owner config already implements ```Serializable ```.